Exercises/4-count-types.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = typeArray => {
+  const hash = {};
+  for (const value of typeArray) {
+    const t = typeof(value);
+    hash[t] = t in hash ? ++hash[t] : 1;
```

Comment:
See alternatives https://github.com/HowProgrammingWorks/DataTypes/pull/21#discussion_r324387734

Reply:
is my decision also right?